src/Microsoft.Azure.ServiceBus/Constants.cs

```diff
@@ -40,5 +40,7 @@ static class Constants
         public static readonly int DefaultClientPumpPrefetchCount = 5;
 
         public static readonly TimeSpan NoMessageBackoffTimeSpan = TimeSpan.FromSeconds(5);
+
+        public static readonly TimeSpan MessageReceiverStartPumpInitialReceiveTimeout = TimeSpan.FromSeconds(1);
```

Comment:
Oh I see. Let me see what that issue is. 
I have a feeling 1 second might be too small. But I don't have a better suggestion as well. Lets leave it at 1 second for now and i'll update it if we come up with a better value.